docs/notebooks/running_test_application.ipynb

```diff
@@ -0,0 +1,354 @@
+   "metadata": {},
+   "source": [
+    "Set paths for\n",
+    "- input_folder (location with stored Sentinel-1 SLC data)\n",
+    "- output_folder (location where processed data will be stored)\n",
+    "- gpt_loction (gpt is located in the bin folder of your SNAP installation)' "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "input_folder = path\n",
+    "output_folder = path\n",
+    "gpt_location = os.path.expanduser('~/snap/bin/gpt')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Create config file with information about input, output and gpt loction"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import yaml\n",
+    "\n",
+    "with open('sample_config_file.yaml') as stream:\n",
+    "   data = yaml.safe_load(stream)\n",
+    "\n",
+    "data['input_folder'] = input_folder\n",
+    "data['output_folder'] = output_folder\n",
+    "data['gpt'] = gpt_location\n",
+    "\n",
+    "with open('test_config_file.yaml', 'wb') as stream:\n",
+    "   yaml.safe_dump(data, stream, default_flow_style=False, \n",
+    "                  explicit_start=True, allow_unicode=True, encoding='utf-8')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Start pre-processing steps"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "from sar_pre_processing.sar_pre_processor import *\n",
+    "import warnings\n",
+    "warnings.filterwarnings(\"ignore\")\n",
+    "\n",
+    "processing = SARPreProcessor(config='test_config_file.yaml')\n",
+    "processing.create_processing_file_list()\n",
+    "print('start step 1')\n",
+    "processing.pre_process_step1()\n",
+    "print('start step 2')\n",
+    "processing.pre_process_step2()\n",
+    "print('start step 3')\n",
+    "processing.pre_process_step3()\n",
+    "print('start add netcdf information')\n",
+    "processing.add_netcdf_information()\n",
+    "print('start create netcdf stack')\n",
+    "processing.create_netcdf_stack()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 3. View processed data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Load netcdf file with processed data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "print(os.getcwd())\n",
+    "print(output_folder)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from netCDF4 import Dataset\n",
+    "import numpy as np\n",
+    "\n",
+    "my_example_nc_file = os.path.join(output_folder + 'data.nc')\n",
```

Comment:
In my notebook, I had to change this. Maybe because I did not state my output directory correctly? It ended with '/test', without a final slash. 

```suggestion
    "    "my_example_nc_file = os.path.join(output_folder + '/test.nc')\n",
",
```